Google.PowerShell/Common/ActiveUserToken.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Google.Apis.Util;
+using System;
+
+namespace Google.PowerShell.Common
+{
+    /// <summary>
+    /// OAuth 2.0 model for a successful access token response as specified in 
+    /// http://tools.ietf.org/html/rfc6749#section-5.1.
+    /// </summary>
+    public class ActiveUserToken
+    {
+        /// <summary>
+        /// The current active user that corresponds to this token.
+        /// </summary>
+        private string activeUser = CloudSdkSettings.GetSettingsValue("account");
```

Comment:
Reading from the Cloud SDK settings here, and again in `IsExpired` too tightly couples the Cloud SDK's implementation and what would otherwise be a straight-forward data object.

Could we instead have `ActuiveUserToken` take an `Action<string>` object (delegate) to compute the current user name on demand? That would make it possible to test `ActiveUserToken` logic without actually accessing Cloud SDK configuration files on disk.

As for complicating usage, IMHO it isn't too bad. And you can always include a static factory method for the common case.

e.g. 
````
public static ActiveUserToken NewCloudSdkUserToken() {
    return new ActiveUserToken(() => CloudSdkSettings.GetSettingsValue("account"));
}
````